### ISSUE TITLE
Option to change text

### ISSUE THREAD

**Issue by roniemartinez** [inventare/django-image-uploader-widget#77]
Thanks for building this awesome library. 🚀 

Is it possible to customize the text (and icon)? If we want to use camera capture, it is still showing "Drop your image or select one" which is not aligned with this custom functionality.

**Comment by EduardoJM** [comment]
Currently, we have two alternatives to change the text of the widget/inline:

- the first is to use the i18n support for change the texts (but that is not a good solution for various different widget uses)
- the second is duplicating the template and overriding the python Widget code (https://github.com/inventare/django-image-uploader-widget/blob/main/image_uploader_widget/widgets.py#L4) or the admin code (https://github.com/inventare/django-image-uploader-widget/blob/main/image_uploader_widget/admin.py#L4).

The second can be used to change the icons too.

For the future (one or two weeks) is possible to add support for those customizations by changing the library code to receive the properties via arguments

**Comment by EduardoJM** [comment]
Hello @roniemartinez,

I created an small update for this library to allow customizating the text and icons into the Widget and Inline itens... That is published into v0.2.2 in the Github (this repo) and pypi (https://pypi.org/project/django-image-uploader-widget/). An small example of how to use it is disponible inside our docs at:

https://inventare.github.io/django-image-uploader-widget/docs/customization/text-and-icons/

**Comment by roniemartinez** [comment]
@EduardoJM Thanks for this new update. We are currently using this version.

I was able to change the text and icon, however, I needed to use `mark_safe` to the icon.

```python
class TestCustomForm(forms.ModelForm):
    class Meta:
        ...
        widgets = {
            'image': ImageUploaderWidget(
                empty_icon=mark_safe("<svg ...></svg>"),  # <--
            ),
        }
        ...
```

I think an alternative to this is by adding `|safe` in the template.

**Comment by EduardoJM** [comment]
Fantastic @roniemartinez , added `| safe` at icons rendering and released with in 0.2.3 into the GitHub and the pypi
